src/remediations/remediations.format.js

```diff
@@ -163,13 +163,13 @@ exports.issues = function (plan_id, issues, total, limit, offset, sort) {
     result.data = issues.map(item => {
         const details = {
             id: item.issue_id,
-            description: item.details.description,
+            description: item.details?.description || '',
             resolution: {},
             resolutions_available: item.resolutionsAvailable,
             system_count: item.systems.length,
         };
 
-        if (details.resolution) {
+        if (item.resolution) {
```

Comment:
It looks like `details.resolution` is being set to `{}` right above this so maybe this is supposed to be `item.resolution`?

Reply:
Yeah 🙄...